_includes/members.html

```diff
@@ -1,30 +1,107 @@
 <section id="members">
-    <h2 class="section-heading">Mitglieder</h2>
-    <hr class="primary">
-    <div class="container-fluid">
-        <div class="row no-gutter">
-            {% assign members_sorted = site.data.members | sort: 'order' %}
-            {% for member in members_sorted %}
-            <div class="col-lg-4 col-sm-6">
-                <a href="javascript:;" onclick="memberExtend('{{member.order}}')" class="portfolio-box">
-                    {% if member.image %}
-                    <img src="{{ member.image }}" class="img-responsive member-image" alt="{{member.name}}" title="{{member.name}}">
-                    {% else %}
-                    <img src="data:image/svg+xml;charset=utf-8,%3Csvg xmlns%3D'http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg' viewBox%3D'0 0 650 350'%2F%3E" class="img-responsive member-image" alt="leeres Bild">
-                    {% endif %}
-                    <div class="portfolio-box-caption">
-                        <div class="portfolio-box-caption-content">
-                            {% if member.type %}<div class="project-category text-faded">{{member.type}}</div>{% endif %}
-                            <div class="project-name">{{member.name}}</div>
-                        </div>
-                    </div>
-                </a>
-                <div class="project-extended" id="{{member.order}}">
-                    {% if member.description %}<p>{{member.description}}</p>{% endif %}
-                    <a href="{{ member.link }}" class="btn btn-default btn-xl" title="{{member.name}}">Mehr erfahren</a>
+  <h2 class="section-heading">Die Vereine</h2>
+  <hr class="primary">
+  <div class="container-fluid">
+    <div class="row">
+      {% assign members_sorted = site.data.members | sort: 'order' %}
+      {% for member in members_sorted %}
+      <div class="col-lg-4 col-12 p-3">
+        <div class="card shadow rounded h-100">
+          {% if member.image %}
+          <img src="{{ member.image }}" class="img-responsive member-image" alt="{{member.name}}" title="{{member.name}}">
+          {% else %}
+          <img src="data:image/svg+xml;charset=utf-8,%3Csvg xmlns%3D'http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg' viewBox%3D'0 0 650 350'%2F%3E" class="img-responsive member-image" alt="leeres Bild">
+          {% endif %}
+          <div class="card-img-overlay d-flex flex-column">
+            <div class="d-flex justify-content-end align-self-end mt-auto">
+              <div class="row pr-3">
+                <div class="col p-1">
+                  <a tabindex="0"
```

Comment:
The info button does not work for me :(

Reply:
Same problem gehe morgen mal die commits zurück ab wann es wieder funktioniert. Eventuell Fullscreen-Card oder Carousel drüberlegen anstattdessen?

Reply:
Somehow the info button started working since Bootstrap 3 was removed.